src/Lean/Data/Lsp/Basic.lean

```diff
@@ -87,6 +99,20 @@ structure TextEdit where
   range : Range
   /-- The string to be inserted. For delete operations use an empty string. -/
   newText : String
+  /-- If this field is present and the editor supports it,
+  `newText` is ignored
+  and an interactive snippet edit is performed instead.
+
+  This is a Lean-specific extension to the LSP standard,
```

Comment:
After reading up more on this, I think it's most accurate to say:
```suggestion
  The use of snippets in `TextEdit`s is a Lean-specific extension to the LSP standard,
  which only provides for snippets in completion items,
```

Reply:
Is this sentence missing a word? It doesn't seem to make sense as it is.

Reply:
Oh huh, I didn't realize that snippets were already supported in completions but not code actions. That's weird! But what you wrote is what I meant by "This" anyway, so may as well disambiguate.